winter/pagination/__init__.py

```diff
@@ -3,3 +3,8 @@
 from .serializer import PageSerializer
 from .page_position_argument_resolver import PagePositionArgumentResolver
 from .page_position_argument_inspector import PagePositionArgumentInspector
+
+
+def setup():
+    from . import page
+    page.setup()
```

Comment:
Maybe inline this call? Just to reduce depth. We are already in the pagination package.
Or move `page.setup()` to `page_inspector.py`, because page itself doesn't care about schema.

Reply:
done